deno/payloads/v10/channel.ts

```diff
@@ -51,7 +51,11 @@ export type TextChannelType =
 	| ChannelType.GuildVoice;
 
 export type GuildChannelType = Exclude<
-	TextChannelType | ChannelType.GuildVoice | ChannelType.GuildStageVoice | ChannelType.GuildAnnouncement,
+	| TextChannelType
```

Comment:
Couldn't this just do Exclude<ChannelType, AllTheDMChannelsTypes>?

Reply:
Oh, yes. That makes more sense